projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-post-v1-1-endpoint.php

```diff
@@ -63,33 +63,51 @@ public function callback( $path = '', $blog_id = 0, $post_id = 0 ) {
 
 		$args = $this->query_args();
 
-		$site = $this->get_platform()->get_site( $blog_id );
-
 		if ( str_contains( $path, '/posts/slug:' ) ) {
+			$site = $this->get_platform()->get_site( $blog_id );
```

Comment:
I am assuming this moved because we don't use `$site` in additional contexts/elsewhere in this API?

Reply:
Yes -- prior to moving all the other logic to `fetch_post()` we did have other uses of `$site`, but now that we've moved that code, the only remaining place we need `$site` is inside this if block.